src/workerd/api/sql-test.js

```diff
@@ -0,0 +1,81 @@
+function assertEqual(a, b) {
+  if (a !== b) {
+    throw new Error(a + " !== " + b);
+  }
+}
```

Comment:
FWIW, if you enable the `nodejs_compat` flag for the test you can `import * as assert from 'node:assert'` and make use of those.